src/templates/template.cpp

```diff
@@ -621,8 +634,19 @@ bool Template::trySetTemplateGeoreferenced(bool /*value*/, QWidget* /*dialog_par
 void Template::applyTemplateTransform(QPainter* painter) const
 {
 	painter->translate(transform.template_x / 1000.0, transform.template_y / 1000.0);
+	// Rotate counter-clockwise.
 	painter->rotate(-transform.template_rotation * (180 / M_PI));
-	painter->scale(transform.template_scale_x, transform.template_scale_y);
+
+	// Scale
+	if (qFuzzyIsNull(transform.template_shear))
+		painter->scale(transform.template_scale_x, transform.template_scale_y);
```

Comment:
Please add parentheses around this line, for symmetry with the `else` branch.

Reply:
Added braces.